ops/review-deploy.tmpl.yaml

```diff
@@ -0,0 +1,220 @@
+replicaCount: 2
+
+livenessProbe:
+  enabled: false
+readinessProbe:
+  enabled: false
+
+brandingVolume:
+  storageClass: nfs
+derivativesVolume:
+  storageClass: nfs
+uploadsVolume:
+  storageClass: nfs
+
+extraVolumeMounts:
+  - name: uploads
+    mountPath: /app/samvera/hyrax-webapp/tmp/imports
+    subPath: imports
+  - name: uploads
+    mountPath: /app/samvera/hyrax-webapp/tmp/exports
+    subPath: exports
+  - name: uploads
+    mountPath: /app/samvera/hyrax-webapp/public/system
+    subPath: public-system
+  - name: uploads
+    mountPath: /app/samvera/hyrax-webapp/public/uploads
+    subPath: public-uploads
+  - name: uploads
+    mountPath: /app/samvera/hyrax-webapp/tmp/network_files
+    subPath: network-files
+
+ingress:
+  enabled: true
+  hosts:
+    - host: hyku-$CI_MERGE_REQUEST_ID.example.com
+      paths:
+        - path: /
+    - host: "*.hyku-$CI_MERGE_REQUEST_ID.example.com"
```

Comment:
does this work? maybe we just want `"*.hyku.example.com"` for an example?

i don't think this values file will succeed at resolving these environment variables.

Reply:
` - envsubst < ops/dev-deploy.tmpl.yaml > ops/dev-deploy.yaml` means yes it will resolve variables. That's why I renamed them tmpl (this matches the nginx and several other significant dockerize projects do things)

I'd be open to moving that to the helm_deploy if you think that is useful

Reply:
ah. this makes sense. ty!